README.md

```diff
@@ -1,6 +1,5 @@
 This is an attempt at recreating an old illustration.  See it live at
-https://numenta.github.io/cell-viz/ or fiddle with it at
-http://jsfiddle.net/7tbm3mv1/2/
+https://numenta.github.io/cell-viz/.
```

Comment:
This jsfiddle no longer applies. We can add another once development settles down.